authors/list_latex.py

```diff
@@ -7,7 +7,11 @@
 author_list = load(open("authors.json"))
 
 with open("../authors.tex", "wb") as f:
```

Comment:
yes, it does!! Thanks a lot. Much better than messing with /usr/lib/python2.7/sitecustomize.py